docs/deploying_apps/deploying_node_js.md

```diff
@@ -19,8 +19,6 @@ These steps assume you have already carried out the setup process explained in t
 
 3. Include an npm ``package.json`` file to specify dependencies. The file should also specify a `start` command used to launch the app.
   
-    The presence of this file is essential to have the application detected as Node.js.
```

Comment:
If you look in `https://github.com/cloudfoundry/nodejs-buildpack/blob/master/bin/detect` - it really does check for package.json presence - as only thing to detect app is node.js